test/com/google/enterprise/adaptor/ad/MockLdapContext.java

```diff
@@ -63,6 +63,11 @@ public MockLdapContext addSearchResult(String filter,
     return this;
   }
 
+  public MockLdapContext clearSearchResults() {
```

Comment:
Why the fluent-style signature, which you aren't using?

Reply:
Keeping the pattern of all the other fluent methods.